src/todo.ts

```diff
@@ -32,6 +33,21 @@ export const removeLicense = async (license: Licence) => {
   }
 };
 
+export const upgradeFirmware = async (firmwareUrl: string) => {
```

Comment:
Should we add this (and line 160 fix) to Vladimir's work as well?

Reply:
line 160 for sure but I already told him about it. This i'm not sure it shows dealing with firmware upgrades who were not done via commands. not all the api is implemented in both of the repos